### PR TITLE
list-component added

### PR DESCRIPTION
**Pull request by buzzsquez**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,14 +1,16 @@
...
+export function App() {
   return (
-    <div className="App">
-      <CharacterList />
-      <CharacterDescription />
-    </div>
+    <BrowserRouter>
+      <Routes>
+        <Route path="/" element={<HomePage />} />
+        <Route path="/characterList" element={<CharacterList />} />
```
Comment (NikitaPFE) [inline src/App.js:11]: dont use camelcase in url
character
character/:id

File: src/pages/CharacterDetails/CharacterDetails.jsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  const characterInfoSource = `https://rickandmortyapi.com/api/character/${id}`;
+  const [characterInfo, setCharacterInfo] = useState([]);
+  const navigate = useNavigate();
+
+  useEffect(() => {
+    fetch(characterInfoSource)
+      .then((resp) => resp.json())
+      .then((data) => {
+        setCharacterInfo(data);
+      });
```
Comment (NikitaPFE) [inline src/pages/CharacterDetails/CharacterDetails.jsx:17]: dont forget about catch

File: src/pages/CharacterList/CharacterList.jsx [hunk 3]
```diff
@@ -1,3 +1,50 @@
...
+                image={character.image}
+                gender={character.gender}
+                status={character.status}
+              />
+            </li>
+          );
+        })}
+      </ul>
+      <footer className={styles.footer}>
+        <button
```
Comment (NikitaPFE) [inline src/pages/CharacterList/CharacterList.jsx:39]: use link instead of button

**Review by NikitaPFE — CHANGES_REQUESTED** [review]

File: src/pages/CharacterDetails/CharacterDetails.jsx (hunk 2 above)
Comment (buzzsquez) [reply src/pages/CharacterDetails/CharacterDetails.jsx:17]: Got it! 
Done!

**Review by buzzsquez — COMMENTED** [review]
Fixed the code.

**Review by NikitaPFE — APPROVED** [review]

**Pull request merged**